database/articles.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const express = require('express');
+const expressApp = express();
+const bodyParser = require('body-parser');
+const jsonParser = bodyParser.json();
+
+let articles = [];
+
+const getAllArticles = () => {
+  return articles;
+};
+
+const postArticle = (req, res) => {
+  if (req.body) {
+    articles.push(req.body);
+    res.send({ success: true });
+    console.log(articles);
```

Comment:
`console.log` should always be removed from PR(s) after debugging. (i know you're seeking feedback so it's fine, just mentioning)